src/main/scala/com/atomist/tree/marshal/LinkedJsonTreeDeserializer.scala

```diff
@@ -103,6 +102,21 @@ object LinkedJsonTreeDeserializer extends LazyLogging {
     }
 }
 
+class EmptyLinkableContainerTreeNode() extends ContainerTreeNode {
+
+  override def nodeName: String = "empty"
+
+  override def value: String = ""
```

Comment:
Thanks for the input both! I'm returning null now. Also added a test to cover a path expression that results in an empty node and None plan.